examples/uct_hello_world.c

```diff
@@ -346,19 +346,24 @@ static ucs_status_t dev_tl_lookup(const cmd_args_t *cmd_args,
         for (md_index = 0; md_index < component_attr.md_resource_count; ++md_index) {
             status = uct_md_config_read(components[cmpt_index], NULL, NULL,
                                         &md_config);
-            CHKERR_JUMP(UCS_OK != status, "read MD config",
-                        release_component_list);
+            if (UCS_OK != status) {
+                fprintf(stderr, "Failed to read MD config: %s\n",
+                        component_attr.md_resources[md_index].md_name);
+                continue;
+            }
 
             status = uct_md_open(components[cmpt_index],
                                  component_attr.md_resources[md_index].md_name,
                                  md_config, &iface_p->md);
             uct_config_release(md_config);
-            CHKERR_JUMP(UCS_OK != status, "open memory domains",
-                        release_component_list);
+            if (UCS_OK != status) {
```

Comment:
Seems such changes are needed in several places, see also pipeline (e,g `test_md`).